editor-components/header/header5/header5.tsx

```diff
@@ -14,32 +26,61 @@ class Header5 extends BaseHeader {
         {
           type: "string",
           key: "titleColored",
-          displayer: "A",
+          displayer: "Subtitle",
           value: "Motivation",
         },
         {
           type: "string",
           key: "title",
           displayer: "Title",
-          value: "Simple Way to Boost Your Creativity and Spark Innovation",
+          value: "THE ART OF FOOD",
         },
         {
           type: "string",
           key: "description",
           displayer: "Description",
-          value: "Looking to enhance your creative thinking skills and spark innovative ideas? In this blog post, we'll explore a simple way to boost your creativity that you can start implementing today.",
+          value: "PHOTOGRAPHY",
         },
+
         {
-          type: "string",
-          key: "buttonText",
-          displayer: "Button Text",
-          value: "Go There",
+          type: "boolean",
+          key: "overlay",
+          displayer: "Overlay",
+          value: true,
         },
         {
-          type: "page",
-          key: "link",
-          displayer: "Button Link",
-          value: "",
+          type: "image",
+          key: "backgroundImage",
+          displayer: "Background Image",
+          value:
+            "https://static.wixstatic.com/media/f61af8_97a68d95c3584eee8952b7b3184f5f80~mv2_d_6924_3840_s_4_2.jpg/v1/fill/w_1699,h_1125,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/f61af8_97a68d95c3584eee8952b7b3184f5f80~mv2_d_6924_3840_s_4_2.jpg",
+        },
+      ],
+    });
+    this.addProp({
+      type: "array",
+      key: "buttons",
+      displayer: "Buttons",
+      value: [
+        {
+          type: "object",
+          key: "button",
+          displayer: "Button",
+          value: [
+            {
+              type: "string",
+              key: "buttonText",
+              displayer: "Button Text",
+              value: "View More",
+            },
+
+            {
+              type: "page",
+              key: "link",
+              displayer: "Button Link",
+              value: "",
+            },
+          ],
         },
```

Comment:
this is an array of objects. please write a type for this.
[check here](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-approach-props)